.gitignore

```diff
@@ -13,3 +13,6 @@ cache
 
 # db
 *.db
+
+# Packages in editable mode
+src
```

Comment:
Do not keep track on git of dependency checked out in editable mode (they will go under `src` folder)

Reply:
@atineoSE can you please clarify what is/are the command(s) you run to generate the `src` directory?

Reply:
@abrichr this was generated when importing the pynput dependency as editable framework. The pynput package is placed under `./src/pynput`